### PR TITLE
Feat: Add Video Listing Page & Video Player Page

### PR DESCRIPTION
**Pull request by Shailseen**
**In this `PR` need to review:**

- Add Card component
- Add Video Listing page
- Add Video Player page
- Maintain Responsiveness
- Add a feature `Tool-Tip` of one card can be open at a time.
[Demo](https://deploy-preview-2--starlit-basbousa-8aa4b7.netlify.app/)

### CODE REVIEW

File: src/context/videos-context.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  useEffect(async () => {
+    try {
+      const response = await axios.get(VIDEOS_API);
+      setVideos(response.data.videos);
+    } catch (error) {
+      console.log(error);
+    }
+  }, []);
+
+  // const getVideosByCategory = (category) => {
```
Comment (jeetbhalani15) [inline src/context/videos-context.js:23]: remove comments

**Review by jeetbhalani15 — APPROVED** [review]
Code looks clean remove comments code rest look good

File: src/context/videos-context.js (hunk 1 above)
Comment (Shailseen) [reply src/context/videos-context.js:23]: okay

**Review by aniketsingh13 — APPROVED** [review]
code structure look good
ui look good
code is easy to read

**Pull request merged**